README.md

```diff
@@ -34,6 +34,42 @@ To stop the docker environment, run:
   docker-compose down
   ```
 
+## Adding a rule
+
+To perform a very basic end-to-end test of StackStorm, let's create a simple rule.
+Run the following from your docker host.
+
+```
+cp -R examples packs.dev
+```
+
+First, let's run `st2 login`:
+
+```
+docker exec eval_st2_1 st2 login -w -p Ch@ngeMe st2admin
+```
+
+We need to tell the FileWatchSensor to watch `/tmp/date.log`, enable the
+`linux.FileWatchSensor` and then call `st2ctl reload`.
+
+```
+docker exec eval_st2_1 sh -c 'echo "    - /tmp/date.log" >> /opt/stackstorm/packs/linux/config.yaml'
```

Comment:
Who would do this? I would `docker exec -it /bin/bash` and have fun there. Can't imagine someone typing the `docker exec eval_st2_1 sh -c` every time.

On bash: so many of our stuff rely on bash and we didn't test it on `sh` that I would be careful.